lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -35,20 +41,22 @@ class CampaignSlide extends React.Component {
           <div className="wrapper">
             {this.renderHeader()}
 
-            {this.campaigns.map(campaign => {
-              // Fix for local files / data missing by preventing null error
-              const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
+            <div className="gallery -triad">
+              {this.state.campaigns.map(campaign => {
+                // Fix for local files / data missing by preventing null error
+                const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
 
-              const props = {
-                id: campaign.id,
-                image_uri: image_uri,
-                tagline: campaign.tagline,
-                title: campaign.title,
-                signup: this.controller.signup.bind(this.controller)
-              }
+                const props = {
+                  id: campaign.id,
+                  image_uri: image_uri,
+                  tagline: campaign.tagline,
+                  title: campaign.title,
+                  signup: this.controller.signup.bind(this.controller)
+                }
 
-              return <CampaignSignupCard {...props} key={campaign.id} />
-            })}
+                return <CampaignSignupCard {...props} key={campaign.id} />
```

Comment:
Yeah, I don't have _strooooong_ feels on it, but I think it looks better stylistically & then we're being explicit about exactly what items we're passing to a component.